vf-react/src/js/components/layout/Apply.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
```

Comment:
This Component is the big Green Apply buttons in the modal popup. Its purpose is to monitor the modal's updates, and alert its ancestor Modules of a change onApply()